### ISSUE TITLE
Can't find yandex-cloud/yandex provider

### ISSUE THREAD

**Issue by daniil-ushkov** [cloudquery/cloudquery#165]
I try to run `cloudquery fetch` on `config.hcl` generated by `cloudquery init yandex-cloud/yandex`:
```
// Configuration AutoGenerated by CloudQuery CLI
cloudquery {
  plugin_directory = "./cq/providers"
  policy_directory = "./cq/policies"

  provider "yandex-cloud/yandex" {
    source  = ""
    version = "latest"
  }

  connection {
    dsn = "host=localhost user=postgres password=pass database=postgres port=5432"
  }
}

// All Provider Configurations

provider "yandex" {
  configuration {
    // Optional. Filter as described https://cloud.yandex.com/docs/resource-manager/grpc/folder_service#List
    // folder_filter = ""
    // Optional. If not specified either using all folders accessible.
    // folder_ids = [<CHANGE_THIS_TO_YOUR_FOLDER_ID>]
    cloud_id = "<my_cloud_id>"
  }
  resources = [
    "compute.addresses",
    "compute.disks",
    "compute.images",
    "compute.instances",
    "iam.service_accounts",
    "kms.keys",
    "vpc.addresses",
    "vpc.networks",
    "vpc.subnets"
  ]
}
```
I get:
```
Starting provider fetch...

Error: no such provider yandex. plugin might be missing from directory or wasn't downloaded
```

Also, i tried to replace "yandex" with "yandex-cloud/yandex" in provider section of config file. Unfortunately, it falls with panic.

Looks like there is a confusion in name of provider ("yandex" or "yandex-cloud/yandex").

**Comment by daniil-ushkov** [comment]
I tried to fix #166

**Comment by roneli** [comment]
Hi,  @daniil-ushkov 

I will take a look at the fix.

Currently to executed a community provider, update the source field to the yandex-cloud org like so:
`
  provider "yandex-cloud/yandex" {
    source  = "yandex-cloud"
    version = "latest"
  }`

I will add to the documentation about this field.

**Comment by roneli** [comment]
Fixed in [v0.14.2](https://github.com/cloudquery/cloudquery/releases/tag/v0.14.2)

**Comment by mirtov-alexey** [comment]
@roneli hello!
could you tell me please what about new syntaxix in yaml?
i have this config and its not working..

how can we add yandex-cloud provider in clouquery init command?
or at minimum make it works with custom config?

`root@opensearch2:~# rm cloudquery.yml 
root@opensearch2:~# vim cloudquery.yml
root@opensearch2:~# ./cloudquery fetch --enable-console-log -v
5:12PM INF logging configured consoleLog=true fileLogging=true fileName=cloudquery.log instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 jsonLogOutput=false logDirectory=. maxAgeInDays=3 maxBackups=3 maxSizeMB=30 verbose=true
5:12PM INF limits (before adjustment) hard_ulimit=1048576 instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 soft_ulimit=1024
5:12PM INF adjusting soft ulimit instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 new_ulimit=16384 previous_ulimit=1024
5:12PM INF limits (after adjustment) hard_ulimit=1048576 instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 soft_ulimit=16384
5:12PM INF Anonymous telemetry collection and crash reporting enabled. Run with --no-telemetry to disable, or check docs at https://docs.cloudquery.io/docs/cli/telemetry instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM INF Invocation parameters args=[] command="cloudquery fetch" core_version=0.32.9 flag:enable-console-log=true flag:verbose=true instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 pflag:enable-console-log=true pflag:verbose=true
5:12PM DBG found existing provider instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 provider=aws version=v0.13.18
5:12PM DBG found existing provider instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2 provider=aws version=v0.13.19
5:12PM DBG adding migration file file=3_v0.24.0.up.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=4_v0.24.1.down.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=4_v0.24.1.up.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=1_v0.19.2.down.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=1_v0.19.2.up.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=2_v0.19.3.down.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=2_v0.19.3.up.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
5:12PM DBG adding migration file file=3_v0.24.0.down.sql instance_id=ec01d920-34fc-4dd4-99ad-fccbb512aff2
Error: invalid provider name "yandex-cloud/yandex-cloud/yandex"`
